tangelo/tangelo/server.py

```diff
@@ -541,6 +541,10 @@ def dirlisting(dirpath, reqpath):
         return result
 
     def execute_analysis(self, query_args):
+        # Hide the identity/version number of the server technology in the
+        # response headers.
+        cherrypy.response.headers["Server"] = "Tangelo"
```

Comment:
Perhaps if security is the root cause of this, even reporting that this is Tangelo is too much.